flink-test-utils-parent/flink-test-utils-junit/src/main/java/org/apache/flink/util/DockerImageVersions.java

```diff
@@ -45,4 +45,6 @@
     public static final String PULSAR = "apachepulsar/pulsar:2.8.0";
 
     public static final String CASSANDRA_3 = "cassandra:3.0";
+
+    public static final String MINIO = "minio/minio:edge";
```

Comment:
what stability does "edge" provide is?

Reply:
`minio/minio:edge` seems to be an old image version which caused weird errors with the hadoop s3 filesystem. I updated the tag to the most recent release which resolved the issue (@rmetzger used this old image in his test as well). That explains why we observed the same error with the s3 hadoop file system.

Reply:
I looked into the versioning again. `edge` seems to be a development version:
```
docker run --entrypoint="" \
  -p 9000:9000 \
  -p 9001:9001 \
  -v /tmp/minio/data2:/data \
  -e "MINIO_ROOT_USER=minio" \
  -e "MINIO_ROOT_PASSWORD=minio123" \
  -it minio/minio:edge /opt/bin/minio --version
minio version DEVELOPMENT.2021-11-23T00-07-23Z
```
It includes a bug that causes the `getFileStatus` to fail on an empty directory to fail. The observed behavior is also present in the released version `RELEASE.2021-11-24T23-19-33Z` but fixed in the succeeding version `RELEASE.2021-12-10T23-03-39Z`

Reply:
does that mean all this was caused by minio being buggy?